src/gimelstudio/core/output_node.py

```diff
@@ -38,12 +40,33 @@ def NodeMeta(self):
         }
         return meta_info
 
+    def NodeInitProps(self):
+        self.export_button = api.ActionProp(
+            idname="export",
+            label="Export",
+            action=self.OnExportButtonPressed
+        )
+        self.NodeAddProp(self.export_button)
+
     def NodeInitParams(self):
         p = api.RenderImageParam('Image', 'Image')
         self.NodeAddParam(p)
 
     def NodeEvaluation(self, eval_info):
         pass
 
+    def OnExportButtonPressed(self, event):
+        # TODO: This is messy. Need to find a cleaner solution to accessing the renderer
+        app_frame = self.nodegraph.parent.parent
+        image = app_frame.renderer.GetRender()
+        try:
+            export_handler = ExportImageHandler(app_frame, image.Image("oiio"))
```

Comment:
The first param *can* be ``None``

Reply:
My thought here was that I've already set the ``app_frame`` variable so I thought I may as well use it here